_plugins/README.md

```diff
@@ -25,7 +25,7 @@ This renders the definition of the glossary term inside a `<div>`, preserving Ma
 
 | Name | Default | Description |
 | --- | --- | --- |
-| `term_id` | N/A (Required) | The `id` of the glossary term whose definition will be used. (This `id` is the same as the filename of the term, i.e. `_data/glossary/<ID>.yml`.) |
+| `term_id` | Not Available (Required) | The `id` of the glossary term whose definition will be used. (This `id` is the same as the filename of the term, i.e. `_data/glossary/<ID>.yml`.) |
```

Comment:
RIght idea, wrong words - The original intention was “not applicable”, meaning “there is no default value here, it’s a required value”. “Not Available” implies that the option isn’t available, which is not what we want to share